HW0/P6/P6.txt

```diff
@@ -1 +1,11 @@
 # Your answers here
+In each parallel run, the order to run the four function in parallel is random. For example, the order of 0,1,2,3 can be random, it could be 1,2,3,0
+For example, we have multiple steps to finish a calculation, and we also have several sub-steps in each steps. If the steps need to be finished step by step, and the sub-steps is independent of other sub-steps in each step. Then we can run sub-steps in parallel, while we mustr run steps in series but not in parallel.
+Otherwise, we would get total wrong results if we run steps in parallel.
+
```

Comment:
I believe that they do run in the same order they were submitted, but they take different times to be done and that is why you get the outputs in a different order.